src/data/nat/parity.lean

```diff
@@ -235,6 +235,14 @@ by { rintro ⟨c, rfl⟩, simp [pow_mul] }
 theorem neg_one_pow_of_odd : odd n → (-1 : R) ^ n = -1 :=
 by { rintro ⟨c, rfl⟩, simp [pow_add, pow_mul] }
 
+lemma even.neg_one_pow_add_right {m n : ℕ} (hm : even m) :
+  (-1 : R) ^ (n + m) = (-1 : R) ^ n :=
+by rw [pow_add, neg_one_pow_of_even hm, mul_one]
+
+lemma even.neg_one_pow_add_left {m n : ℕ} (hm : even m) :
+  (-1 : R) ^ (m + n) = (-1 : R) ^ n :=
+by rw [add_comm, even.neg_one_pow_add_right hm]
```

Comment:
Also, should the `neg_one_pow` stuff be tagged with `@[parity_simps]`? It's kind of niche I guess, but you have to manually specify `with parity_simps` anyways.

Reply:
I think the `parity_simps` simp set is for simplifying `even x` and `odd x` expressions (for various expressions `x`), not for general theorems predicated on `even x` and `odd x`. So I would say leave it as is, although I can see a decent argument in the other direction.